src/components/layouts/nav-user.tsx

```
@@ -0,0 +1,104 @@
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                <IconNotification />
+                Notifications
+              </DropdownMenuItem>
+            </DropdownMenuGroup>
+            <DropdownMenuSeparator />
+            <DropdownMenuGroup className='w-full'>
+              <DropdownMenuItem className='w-full cursor-pointer'>
+                <span className='w-full'>
+                  <SignOutButton>
+                    <button className='w-full flex items-center gap-2'>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)